### PR TITLE
Update to nginx:1.20.1-alpine and openjdk/8u302-slim

### PR DESCRIPTION
**Pull request by ljdelight**
Update scalasbt deb URLs

Stop proxying /assets to the maproulette-api container

### CODE REVIEW

**Comment by ljdelight** [comment]
FYI @mvexel

File: api/Dockerfile [hunk 1]
```diff
@@ -18,7 +20,7 @@ ARG CACHEBUST=1
 RUN echo $CACHEBUST
 # Download Maproulette V2
 RUN git clone https://github.com/${GIT}.git /maproulette-api
-RUN chmod 777 /maproulette-api
+RUN chmod 775 /maproulette-api
```
Comment (ddkpisme) [inline api/Dockerfile:23]: 👍

File: frontend/Dockerfile [hunk 2]
```diff
@@ -23,7 +23,7 @@ ENV NODE_OPTIONS="--max-old-space-size=8192"
 RUN yarn && yarn run build
 
 # Production environment
-FROM nginx:1.15.8-alpine
+FROM nginx:1.20.1-alpine
```
Comment (ddkpisme) [inline frontend/Dockerfile:26]: 🥇

**Review by ddkpisme — APPROVED** [review]
lgtm!

**Comment by ljdelight** [comment]
I had a typo in a `FROM` (a slash instead of a colon) and needed to install `curl` in the updated api image. I didn't catch this because the scripts don't fail when there are errors and the tail end of the build says it was successful. I'll see about getting `set -exu` added to avoid this in the future

**Comment by ljdelight** [comment]
@mvexel does it look good to merge?

**Review by mgcuthbert — APPROVED** [review]

**Pull request merged**